package.json

```diff
@@ -21,7 +22,7 @@
 		"prepare": "husky install"
 	},
 	"engines": {
-		"node": ">=20"
+		"node": ">=18"
```

Comment:
And why downgrade to 18?

Reply:
> there is no reason to have 20 here, check the dependencies. npm and pnpm only show a warning, but yarn throws an error

Reply:
You mean use an older version because we don't need anything newer? What's wrong with just using the newest?

Reply:
I'm using 18 and I don't want to upgrade without the reason

Reply:
You can build the Marketplace even with node 12

Reply:
Just use NVM, I probably have like 6 node versions installed